docs/shared-components-reference.md

```
@@ -0,0 +1,749 @@
+                <div className='prose'>{post.content}</div>
+            </article>
+        </ContainerLayout>
+    )
+}
+```
+
+---
+
+## Component Decision Tree
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)